synapse/models/science.py

```diff
@@ -0,0 +1,92 @@
+import synapse.lib.module as s_module
+
+class ScienceModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('sci', {
+            'types': (
+                ('sci:hypothesis:type:taxonomy', {}, {}),
+                ('sci:hypothesis', ('guid', {}), {
+                    'doc': 'A hypothesis or theory.'}),
+
+                # TODO link experiment to eventual procedure node
+                ('sci:experiment:type:taxonomy', {}, {}),
+                ('sci:experiment', ('guid', {}), {
+                    'doc': 'An instance of running an experiment to test a hypothesis.'}),
+
+                ('sci:observation', ('guid', {}), {
+                    'doc': 'An observation which may have resulted from an experiment.'}),
```

Comment:
```suggestion
                    'doc': 'An event that occurred or was seen during an experiment.'}),
```

Reply:
The definition of a form does not require not using the noun in the definition. This is not necessarily an event, an observation is most accurate.